client/src/pages/Report/NodesSection/NodesSection.tsx

```diff
@@ -137,16 +116,22 @@ const NodesSection = () => {
         <SectionComponent
             icon={<SVGIcon iconName="application-icon" />}
             title={'Nodes'}
-            callback={handleAddClick}
-        >
+            callback={() => setShowModal(true)}>
             {showModal && (
-                <OverlayComponent isDisplayed={showModal} onClose={handleCloseModal}>
-                    <p>No nodes here (probably Wojciech dropped all of them)</p>
+                <OverlayComponent
+                    isDisplayed={showModal}
+                    onClose={handleCloseModal}
+                >
+                    <NodesEntriesSelector
+                        selectedNodes={selectedNodes}
+                        setSelectedNodes={setSelectedNodes}
+                        nodesToExclude={rows}
+                        onAdd={handleAddNodes}
+                        onClose={handleCloseModal}
+                    />
```

Comment:
Why using showModal if OverlayComponent already decides whether it is displayed with isDisplayed?